modules/custom/d_p_tiles/js/d_p_tiles.js

```diff
@@ -12,14 +12,14 @@
 
       // Enable Masonry.
       $(document).ready(function () {
-        var $wrapper = $('.d-tiles, .d-side-tiles', context);
```

Comment:
Let's not change it, it will break compatibility with IE, as we do not use Babel. We will do it in the future.

Reply:
@MikolajMendys-droptica this is still to be reverted